packages/inngest/src/components/InngestFunction.ts

```diff
@@ -408,7 +408,7 @@ export namespace InngestFunction {
        * If timeout is reached, the function will be invoked with a batch
        * even if it's not filled up to `maxSize`.
        *
-       * Expects 1s to 60s.
+       * Expects a time string such as 1s, 60s or 1h30m15s.
```

Comment:
```suggestion
       * Expects a time string such as 1s, 60s or 2m
```

I wouldn't use `1h30m15s` as an example since our max appears to be 20 minutes

Reply:
Oh, gotcha....I just copied that from the comment on the `TimeStrBatch` referenced here. I'll knock it down.